python/lsst/ctrl/bps/bps_utils.py

```diff
@@ -191,3 +194,35 @@ def parse_count_summary(summary):
         label, count = part.split(":")
         counts[label] = count
     return counts
+
+
+def dump_pkg_info(filename):
+    """Save information about versions of packages in use for future reference.
+
+    Parameters
+    ----------
+    filename : `str`
+        The name of the file where to save the information about the versions
+        of the packages.
+    """
+    file = Path(filename)
```

Comment:
I'd like us to start using `lsst.resources.ResourcePath` in new code if at all possible.

Reply:
I made a ctrl_bps ticket to make the lsst.resources.ResourcePath changes (DM-32871) that we can start next if no fires.